In the scenario above, an error is supposed to occur because we try to convert characters into integers. Instead of the error, the program caught the exception “NumberFormatException” itself, and printed out that result so we know.

